src/Card.tsx

```diff
@@ -54,7 +54,7 @@ export function Card(props: Props): JSX.Element {
           <MaterialIconButton
             onClick={props.handleClose}
             size="small"
-            sx={{ position: "absolute", top: "10px", right: "7px" }}
+            sx={{ position: "absolute", top: "8px", right: "7px" }}
```

Comment:
<img width="343" alt="Screenshot 2022-05-03 at 10 39 12" src="https://user-images.githubusercontent.com/37402968/166432901-aa09bff2-96b9-46c2-a8cb-5ce2a3698f47.png">

@joshuajames-smith let me know if this is any better. Thanks

Reply:
Great, thank you!